tools/backups2datalad/adandi.py

```diff
@@ -236,3 +252,151 @@ async def apublish(self, max_time: float = 120) -> RemoteDandiset:
 
     async def adelete(self) -> None:
         await self.aclient.delete(self.api_path)
+
+
+class RemoteAsset(BaseModel):
+
+    dandiset: RemoteDandiset
+    #: The asset identifier
+    identifier: str = Field(alias="asset_id")
+    #: The asset's (forward-slash-separated) path
+    path: str
+    #: The size of the asset in bytes
+    size: int
+    #: The date at which the asset was created
+    created: datetime
+    #: The date at which the asset was last modified
+    modified: datetime
+    #: The asset's raw metadata
+    metadata: Dict[str, Any]
+
+    class Config:
+        allow_population_by_field_name = True
+        arbitrary_types_allowed = True
```

Comment:
This configures pydantic so that the class can use the non-pydantic `RemoteDandiset` class as an attribute and so that instances can be contructed with `RemoteAsset(identifier=...)` instead of `RemoteAsset(alias_id=...)` (The latter setting's not used here; the `Config` was just copied from dandi-cli).